exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -390,4 +391,15 @@
 
   String DYNAMIC_UDF_SUPPORT_ENABLED = "exec.udf.enable_dynamic_support";
   BooleanValidator DYNAMIC_UDF_SUPPORT_ENABLED_VALIDATOR = new BooleanValidator(DYNAMIC_UDF_SUPPORT_ENABLED, true, true);
+
+  /**
+   * Option whose value is a long value representing the number of bits required for computing ndv (using HLL)
+   */
+  LongValidator NDV_MEMORY_LIMIT = new PositiveLongValidator("exec.statistics.ndv_memory_limit", 30, 20);
+
+  /**
+   * Option whose value represents the current version of the statistics. Decreasing the value will generate
+   * the older version of statistics
+   */
+  LongValidator STATISTICS_VERSION = new NonNegativeLongValidator("exec.statistics.capability_version", 1, 1);
```

Comment:
Yes. By default, the stats version computed is the latest version. However, if the user chooses to compute an older version (maybe the performance is not optimal) they can set this option to an older version. When planning a query, Drill would interpret the statistics based on the version in the stats file.